packages/zwave-js/src/lib/commandclass/SceneControllerConfigurationCC.ts

```diff
@@ -148,9 +148,27 @@ export class SceneControllerConfigurationCCAPI extends CCAPI {
 					) ?? new Duration(0, "seconds");
 				await this.set(propertyKey, value, dimmingDuration);
 			}
+		} else if (property === "dimmingDuration") {
+			if (typeof value !== "number") {
+				throwWrongValueType(
+					this.ccId,
+					property,
+					"duration",
+					typeof value,
+				);
+			}
+
+			const dimmingDuration = Duration.parseSet(value);
```

Comment:
Morally, I feel like we should throw some kind of error if `parseSet` is unable to parse the `value`, but I don't see an appropriate error to `throw`. I was looking for something like `throwInvalidValue`, but didn't see anything. So, instead I'm just failing through to setting the `dimmingDuration` to `0x00`.

Reply:
IMO we should use `Duration.from` here:
https://github.com/zwave-js/node-zwave-js/blob/17ae97650e93df81e606f47bd5b24fd993c3635b/packages/zwave-js/src/lib/commandclass/BinarySwitchCC.ts#L128
If the result is `undefined`, i.e. could not be parsed, you can do this
```ts
	throw new ZWaveError(
		`${CommandClasses[cc]}: "${property}" could not be set. ${value} is not a valid duration`,
		ZWaveErrorCodes.Argument_Invalid,
	);
```

Reply:
Okay, now I see why `Duration.from` should be used here. I was thinking that the clients would send a `number` representing the encoding of the duration, but it looks like Home-Assistant would send a `string` and zwavejs2mqtt would send a `Duration` object.

However, using `Duration.from` is awkward here, because it requires that `typeof value` is `string | Duration | undefined`, which means that I need to coerce `value` into a `Duration` if it isn't a `string`.